src/Command/ExplainAdminCommand.php

```diff
@@ -118,6 +119,7 @@ public function execute(InputInterface $input, OutputInterface $output)
         }
 
         $metadata = $this->validator->getMetadataFor($admin->getClass());
+        \assert($metadata instanceof ClassMetadata);
```

Comment:
What if it's not a ClassMetadata ? Is it possible ?

Reply:
Same than https://github.com/sonata-project/SonataAdminBundle/pull/6161#discussion_r443250848

I agree we need that this is a `ClassMetadata`.
But if it's theorically possible that we get something else, an exception would be more helpful.
If we're sure the $metadata is a ClassMetadata, an assert is enough.